lib/generators/suspenders/factory_bot_generator.rb

```diff
@@ -0,0 +1,69 @@
+module Suspenders
+  module Generators
+    class FactoryBotGenerator < Rails::Generators::Base
+      source_root File.expand_path("../../templates/factory_bot", __FILE__)
+      desc "Installs and configures factory_bot_rails. Supports the default test suite and RSpec."
```

Comment:
I should use [this description](https://github.com/thoughtbot/suspenders/blob/main/templates/descriptions/factories.md)

Reply:
6e793a9e3a48d387bb3fdb514c8ffab5ec21734c